java/struts2/config.yaml

```diff
@@ -0,0 +1,6 @@
+framework:
+  website: struts.apache.org
+  version: 2.5.22
```

Comment:
could you use `X.Y` format (`2.5`), since it's clear for readers

Reply:
👍